R/expectedDist.R

```diff
@@ -44,7 +44,7 @@ expectedDist <- function(align.stats, position=NULL, offset=NULL, number=100, ge
     combined <- DNAStringSet(paste0(front.seq, back.seq))
     if (is(front.seq, "QualityScaledDNAStringSet")) {
         combined.qual <- paste0(quality(front.seq), quality(back.seq))
-        combined <- QualityScaledDNAStringSet(combined, combined.qual)
+        combined <- QualityScaledDNAStringSet(combined, PhredQuality(combined.qual))
```

Comment:
There are many different qualities (e.g., Solexa, Illumina) - make sure you use the same quality type as the input.

Reply:
Fixed